javascript/boilerplate-text/boilerplate-fr.js

```diff
@@ -67,10 +68,10 @@ s.i18nActivityHomePage = "Page d’accueil de l’activité d’internationalisa
 s.home = "Accueil";
 s.contactI18nActivity = "Pour obtenir de l'aide au sujet de l'internationalisation au W3C.";
 s.contact = "Contacter";
-s.aboutI18nActivity = "About W3C Internationalization."
-s.about = "About"
-s.askI18nActivity = "Ask for help or information."
-s.ask = "Ask"
+s.aboutI18nActivity = "Pour en savoir plus sur l'internationalisation au W3C."
+s.about = "En savoir plus"
```

Comment:
Is this the standard translation for "About" in the sense of "About the W3C i18n Activity"?  It's not really a question of learning more, but specifically about information related to the organisation.  Perhaps something like á propos??  (Just asking)